module/strategy/WalkInsideBoundedBox/src/WalkInsideBoundedBox.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef MODULE_STRATEGY_WALKINSIDEBOUNDEDBOX_HPP
+#define MODULE_STRATEGY_WALKINSIDEBOUNDEDBOX_HPP
+
+#include <Eigen/Core>
+#include <nuclear>
+
+#include "extension/Behaviour.hpp"
+
+namespace module::strategy {
+
+    class WalkInsideBoundedBox : public ::extension::behaviour::BehaviourReactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+
+            /// @brief Bounded box of region to defend the ball - x min, x max, y max, y min
+            Eigen::Vector4f bounded_region        = Eigen::Vector4f::Zero();
+            Eigen::Vector2d bounded_region_xy_min = Eigen::Vector2d::Zero();
```

Comment:
these are never being used

Reply:
They will be when I'm finished. I went to bed after pushing last night